.travis.yml

```diff
@@ -41,6 +41,7 @@ addons:
     sources:
       - sourceline: 'deb https://apt.kitware.com/ubuntu/ xenial main'
         key_url: 'https://apt.kitware.com/keys/kitware-archive-latest.asc'
+      - sourceline: 'deb https://apt.kitware.com/ubuntu/ xenial-rc main'
```

Comment:
Are we planning to use pre-release versions in the near future? If this is a temporary change then might be good to raise an issue on Github and remove this when we don't need it

Reply:
I am not sure what Mbed OS future plans are. When Mbed OS limits themselves to only released versions, we can drop this, yes.

Reply:
https://github.com/ARMmbed/mbed-tools/issues/104